src/main/java/uk/gov/hmcts/darts/transcriptions/component/impl/TranscriberViewSummaryRowMapper.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.darts.transcriptions.component.impl;
+
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.darts.transcriptions.model.TranscriberViewSummary;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.time.LocalDate;
+import java.time.OffsetDateTime;
+
+@Component
+public class TranscriberViewSummaryRowMapper implements RowMapper<TranscriberViewSummary> {
+
+    @Override
+    public TranscriberViewSummary mapRow(ResultSet rs, int rowNum) throws SQLException {
+        var summary = new TranscriberViewSummary(
+            Integer.valueOf(rs.getInt("transcription_id")),
+            Integer.valueOf(rs.getInt("case_id")),
```

Comment:
I think you can eliminate the `Integer.valueOf()` wrapper here, as `getInt` already produces an `int`.

Reply:
Done in commit b2783865